vscode/zoo.code-workspace

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
might be good to .gitignore vscode files if they are just ide config files?

Reply:
I am finding that vscode is such a poor tool that it makes sense to keep the configuration under version control.
vscode in general does not work if the configuration for a repository is outside of the repository, this should not be the case, but then it glitches.
Also, I'm using vscode due to its popularity.  I want to prove the repository is compatible.
I prefer this stays as it is.  Do you agree? mark "resolved" in this case.